src/xunit.performance.execution/BenchmarkEventSource.cs

```diff
@@ -147,45 +145,61 @@ public unsafe void BenchmarkIterationStart(string RunId, string BenchmarkName, i
                 if (RunId == null)
                     RunId = "";
 
+                AllocatedBytes = AllocatedBytesForCurrentThread.LastAllocatedBytes;
                 fixed (char* pRunId = RunId)
                 fixed (char* pBenchmarkName = BenchmarkName)
                 {
-                    EventData* data = stackalloc EventData[3];
+                    const int eventDataCount = 4;
+                    EventData* data = stackalloc EventData[eventDataCount];
                     data[0].Size = (RunId.Length + 1) * sizeof(char);
                     data[0].DataPointer = (IntPtr)pRunId;
                     data[1].Size = (BenchmarkName.Length + 1) * sizeof(char);
                     data[1].DataPointer = (IntPtr)pBenchmarkName;
                     data[2].Size = sizeof(int);
                     data[2].DataPointer = (IntPtr)(&Iteration);
-                    WriteEventCore(3, 3, data);
+                    data[3].Size = sizeof(long);
+                    data[3].DataPointer = (IntPtr)(&AllocatedBytes);
+
+                    WriteEventCore(3, eventDataCount, data);
                 }
             }
         }
 
-        [Event(4, Opcode = EventOpcode.Info, Task = Tasks.BenchmarkIterationStop)]
-        public unsafe void BenchmarkIterationStop(string RunId, string BenchmarkName, int Iteration)
+        [Event(4,
```

Comment:
Nit: Can you put this all on one line?

Also, do you need to specify Level? It is not specified for the other events and wasn't specified for this event until this PR.